### PR TITLE
Tc 3245 marketing site

### PR DESCRIPTION
**Pull request by crstalli**
There may be more clean-up necessary, but I spent a couple hours re-doing things to get this in a state I thought was presentable.

### CODE REVIEW

File: templates/handlebars/thirdchannel/authentication/register_form.hbs [hunk 1]
```diff
@@ -0,0 +1,37 @@
+
```
Comment (neolinkpsu) [inline templates/handlebars/thirdchannel/authentication/register_form.hbs]: Not sure why we have a handlebars template for only the form elements.  We should either put these elements into the html on rails or have the complete form here

File: js/apps/thirdchannel/views/opportunities/main.js [hunk 2]
```diff
@@ -0,0 +1,125 @@
+define(function (require) {
```
Comment (neolinkpsu) [inline js/apps/thirdchannel/views/opportunities/main.js]: Seems to be a copy and paste of the login.js file.  If we've got similar functions being used here, we should create a new class that both can inherit from.

File: js/apps/thirdchannel/views/opportunities/main.js [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+        context = require('context'),
+        templates = require('handlebarsTemplates');
+
+    return Backbone.View.extend({
+        el: '#registration-form-container',
+
+        registrationTemplate: templates['thirdchannel/authentication/register_form'],
+        initialize: function () {
+            anchorTransition(-65, 1000);
+            this.listenTo(this, 'setupValidation', this.validateRegistrationForm);
```
Comment (neolinkpsu) [inline js/apps/thirdchannel/views/opportunities/main.js]: Not understanding why you need this

File: js/apps/thirdchannel/views/opportunities/main.js [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+            this.renderRegistrationForm();
+        },
+
+        displaySignUpForm: function (e) {
```
Comment (neolinkpsu) [inline js/apps/thirdchannel/views/opportunities/main.js]: this and renderRegistrationForm seems like duplicate functions

File: js/apps/thirdchannel/views/opportunities/main.js [hunk 5]
```diff
@@ -0,0 +1,125 @@
...
+            this.renderRegistrationForm();
+        },
+
+        renderRegistrationForm: function () {
+            this.$el.html(this.registrationTemplate());
+            this.trigger('setupValidation');
+        },
+
+
+        validateRegistrationForm: function () {
```
Comment (neolinkpsu) [inline js/apps/thirdchannel/views/opportunities/main.js]: Lets extract this into a class which we can extend

File: css/scss/apps/thirdchannel/_application.scss [hunk 6]
```diff
@@ -34,33 +34,131 @@ form.application-form {
   float:right;
 }
 
-.opportunity-content {
-  .opportunity-header {
+#opportunities-page {
```
Comment (neolinkpsu) [inline css/scss/apps/thirdchannel/_application.scss]: This seems like a lot of CSS for a page that should be using our basic styles.  Lets talk about this when you get in

**Comment by neolinkpsu** [comment]
lgtm

**Comment by crstalli** [comment]
Tested on staging - I'm going to fill out the confluence page and send an email first thing tomorrow before merging to give program manager's some time to prepare the content.

**Pull request merged**